app/data_providers/dispatch.py

```diff
@@ -148,29 +148,31 @@ async def get_slate_lineup(
         """
         slates = []
 
-        user_impression_capped_list, preferred_topics, contentv1_assignment = await gather(
+        user_impression_capped_list, preferred_topics, thompson_sampling_assignment = await gather(
             self.user_impression_cap_provider.get(user),
             self._get_preferred_topics(user),
-            self.unleash_provider.get_assignment('temp.web.recommendation-api.home.contentv1', user=user),
+            self.unleash_provider.get_assignment('temp.web.recommendation-api.home.thompson-sampling', user=user),
```

Comment:
I mentioned live that Unleash has some benefits, so just for future reference:
- We have the ability to create experiments in Unleash without being dependent on another team.
- By using Unleash we hook into the analytics ecosystem, which allows us to measure impact on Pocket metrics, such as retention.
- Unleash can easily make experiments sticky to the user id. Less code we need to write in recommendation-api.

That said, we can easily switch to another experimentation framework if needed because our code is modular.